setup.py

```diff
@@ -367,7 +367,7 @@ def build_extensions(self):
                     }
 
                 for platform_ in arch_tp:
-                    dirs = libdirs.get(platform_, None)
+                    dirs = libdirs.get(platform_, libdirs)
```

Comment:
No. 

`libdirs` is a dict, `libdirs[platform_]` is a list. 

Then here:
```
 for path in dirs:
                         _add_directory(library_dirs, path)		                         _add_directory(library_dirs, path)
```
you wind up adding all the keys from the libdirs dict to the path list.